helpers/integration.go

```diff
@@ -423,3 +423,19 @@ func RemoveFinalizerFromPipelineRun(adapterClient client.Client, logger Integrat
 
 	return nil
 }
+
+// AddFinalizerToPipelineRun adds the finalizer to the PipelineRun.
+// If finalizer was not added successfully, a non-nil error is returned.
+func AddFinalizerToPipelineRun(adapterClient client.Client, logger IntegrationLogger, ctx context.Context, pipelineRun *tektonv1beta1.PipelineRun, finalizer string) error {
```

Comment:
The PR checks are failing on this line with the error: `undefined: tektonv1beta1 (compile)`.
I think we should be using the new import `tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"` instead.

Reply:
Fixed.  Also fixed a couple of other outdated references to `tektonv1beta1` that I had also missed earlier.